### ISSUE TITLE
Table of hell: Berlin/de

### ISSUE THREAD

**Issue by xeladotbe** [spencermountain/wtf_wikipedia#150]
https://de.wikipedia.org/w/api.php?action=query&redirects=true&prop=revisions&rvprop=content&maxlag=5&format=json&origin=*&titles=Berlin

wtf(...).sentences(0).text()
`{| class="wikitable toptextcells float-right" style="width:330px; background:#FFFFFF;"`

wtf(...).sentences(1).text()
`! colspan="2" class="hintergrundfarbe5" style="font-size:larger;"| Land Berlin`

**Comment by xeladotbe** [comment]
possible fix:

use of find_recursive:

```
const parseTable = require('./parseTable');
const Table = require('./Table');
const find_recursive = require('../lib/recursive_match');

//tables can be recursive, so looky-here.
const findTables = function(section, wiki) {
  let tables = [];
  let matches = find_recursive('{', '}', wiki);

  matches.forEach(match => {
    if (match.indexOf('{|') !== -1 && match.indexOf('|}') !== -1) {
      wiki = wiki.replace(match, '');

      let data = parseTable(match);

      if (data && data.length > 0) {
        tables.push(new Table(data));
      }
    }
  });

  if (tables.length > 0) {
    section.tables = tables;
  }

  return wiki;
};

module.exports = findTables;

```

**Comment by spencermountain** [comment]
wow, what a table 😳
thanks Alexander. Clever solution.
I think, for whatever reason, i ran away from this idea. I don't remember why. You can see some of the embedded table tests fail when I tried it.
Some of this needs a re-think.

For reference, this isolates the failed table embed:
```js
let str = ` {| class="oopsie"
| first row
|-
| Secod row
{|
|-
| embed 1
|-
| embed 2
|}
|-
| Berlin!
|-
|}

Actual first sentence  is here`;
console.log(wtf(str).sentences(0).text());
```
will take a look at it this week, nice find.

**Comment by spencermountain** [comment]
fixed in 5.1.0
thanks!
